src/components/molecules/VenuePartygoers/VenuePartygoers.tsx

```diff
@@ -12,30 +12,24 @@ export interface VenuePartygoersProps {
 export const VenuePartygoers: React.FC<VenuePartygoersProps> = ({
   venueId,
 }) => {
-  const { isLoading, parentVenue, currentVenue } = useRelatedVenues({
+  const { isLoading } = useRelatedVenues({
     currentVenueId: venueId,
   });
```

Comment:
If no `parentVenue` nor `currentVenue` are needed `useRelatedVenues` can be removed. Little sense to keep it just so `isLoading` can be used.

Reply:
removed it in 0700a1d